### PR TITLE
FirstMonitor

### PR DESCRIPTION
**Pull request by simonepv**
Uploaded fisrt monitor tools for HV and LV in QC8.
BitStatus.txt contains the meaning of the status bits for HV and LV boards

### CODE REVIEW

File: QC8Monitor/BitStatus.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
+HV boards
```
Comment (jshlee) [inline QC8Monitor/BitStatus.txt]: move to Monitoring/data dir

File: QC8Monitor/StatusVmon904HV.py [hunk 2]
```diff
@@ -0,0 +1,667 @@
+import cx_Oracle
```
Comment (jshlee) [inline QC8Monitor/StatusVmon904HV.py]: move to Monitoring/script or Monitoring/python depending on use

File: QC8Monitor/StatusVmon904LV.py [hunk 3]
```diff
@@ -0,0 +1,714 @@
+import cx_Oracle
```
Comment (jshlee) [inline QC8Monitor/StatusVmon904LV.py]: do we really need two different code for this.
its better to make 1 script that does both

**Comment by simonepv** [comment]
Dear Jason,

I made two scripts for shortage of time and because it was indicated to me
that it was ok.
To decide what to do, can you please participate to the meeting of tomorrow
at 11 am?
https://indico.cern.ch/event/808785/

Thank you for your time

Best regards

Simone Calzaferri

Best regards

Simone Calzaferri

Il giorno mer 27 mar 2019 alle ore 07:04 Jason Lee <notifications@github.com>
ha scritto:

**Comment by simonepv** [comment]
Reorganised the Monitoring repository

**Comment by jshlee** [comment]
move to Monitoring/data
not move to Monitoring/datadir

**Comment by jshlee** [comment]
i would prefer 1 script because it reduces bugs and makes it easier to maintain

**Comment by simonepv** [comment]
Dear Jason,

I understand your point. But can you please come to the meeting to discuss
and to define the priorities?
I need to know the priority of this task to organise my work.

Best regards

Simone Calzaferri

Il Mer 27 Mar 2019, 12:52 Jason Lee <notifications@github.com> ha scritto:

**Pull request merged**